en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md

```diff
@@ -112,24 +105,26 @@ CREATE TABLE `Player` (
 
 ## Creating Data Records
 
-A new instance of a [DataObject](api:SilverStripe\ORM\DataObject) can be created using the `new` syntax.
+A new instance of a [DataObject](api:SilverStripe\ORM\DataObject) can be created using the `new` keyword.
 
 ```php
 $player = new Player();
 ```
 
-Or, a better way is to use the `create` method.
+But a better way is to use the `create()` method.
```

Comment:
The word "but" doesn't automatically mean everything written before it is incorrect or untrue.... but I'll make this change to avoid PR ping pong since it doesn't materially change anything.

Reply:
Done